### ISSUE TITLE
Breakfast "idt" feature  mof  performance

### ISSUE THREAD

**Issue by AngxiaoYue** [MohsenFayyaz89/SCT#3]
hi! Thank you for your well orgnised code.
sorry to disturb you! I want to reproduce your results and I use "idt" feature which you also mentioned in the paper. I wonder how I should set the parameter below because I am struggling with it.
_C.training.optimizer=?
_C.training.learning_rate=?
_C.training.momentum = ?
or other important parameters I missed...
Also, I set _C.training.overfit = True which means that test_db=train_db. But in this situation, the loss decrease, mof is only 0.5 or even 0.2 maybe. when _C.training.overfit = False, mof on real test set is only 0.14. may some possible issues I ignored? Looking forward to your reply!
